care/facility/management/commands/clean_patient_phone_numbers.py

```diff
@@ -0,0 +1,38 @@
+import json
+from typing import Optional
+
+import phonenumbers
+from django.core.management.base import BaseCommand
+from phonenumber_field.serializerfields import PhoneNumberField
+from rest_framework.exceptions import ValidationError
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.exceptions'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5332931)